components/automate-ui/src/app/pages/integrations/detail/integrations-detail.component.html

```diff
@@ -35,10 +35,14 @@
 
     <div class="page-body">
       <chef-toolbar>
+        <chef-button primary
+          routerLink="/compliance/scanner/nodes/add">
+          Add nodes
```

Comment:
should this be "Add Nodes" ? https://github.com/chef/automate/issues/138#issuecomment-491063115